middlewares/authzWorkspace.sql

```diff
@@ -3,7 +3,7 @@ SELECT
   v.question_id,
   v.instance_question_id,
   v.course_instance_id,
-  q.course_id,
+  v.course_id,
```

Comment:
I believe this is correct, as we want to authorize based on the consuming course in the case of a shared question.